README.md

```diff
@@ -54,85 +72,70 @@ The Optimism Immunefi program offers up to $2,000,042 for in-scope critical vuln
 ├── <a href="./op-program">op-program</a>: Fault proof program
 ├── <a href="./op-proposer">op-proposer</a>: L2-Output Submitter, submits proposals to L1
 ├── <a href="./op-service">op-service</a>: Common codebase utilities
+├── <a href="./op-ufm">op-ufm</a>: Simulations for monitoring end-to-end transaction latency
 ├── <a href="./op-wheel">op-wheel</a>: Database utilities
+├── <a href="./ops">ops</a>: Various operational packages
 ├── <a href="./ops-bedrock">ops-bedrock</a>: Bedrock devnet work
 ├── <a href="./packages">packages</a>
 │   ├── <a href="./packages/chain-mon">chain-mon</a>: Chain monitoring services
 │   ├── <a href="./packages/common-ts">common-ts</a>: Common tools for building apps in TypeScript
-│   ├── <a href="./packages/contracts-ts">contracts-ts</a>: ABI and Address constants
 │   ├── <a href="./packages/contracts-bedrock">contracts-bedrock</a>: Bedrock smart contracts
+│   ├── <a href="./packages/contracts-ts">contracts-ts</a>: ABI and Address constants
 │   ├── <a href="./packages/core-utils">core-utils</a>: Low-level utilities that make building Optimism easier
+│   ├── <a href="./packages/fee-estimation">fee-estimation</a>: Tools for estimating gas on OP chains
 │   └── <a href="./packages/sdk">sdk</a>: provides a set of tools for interacting with Optimism
+│   └── <a href="./packages/web3js-plugin">web3js-plugin</a>: Adds functions to estimate L1 and L2 gas
+├── <a href="./patches">patches</a>: Is this used somewhere or should it be deleted?
```

Comment:
Is this folder still used? If so, I couldn't find where, and we should update the summary copy here. If it's not used, I'll delete this

Reply:
So `patches` contains a patch that is applied to the JS dep inside. To be honest I have no idea if we need this patch anymore, it is for `changesets-cli`. We were using changesets to release code other than JS so its possible that we no longer need it now that we do releases outside of changesets for JS code. I haven't put in the effort to learn if we can delete it because I didn't want to accidentally break JS releases

Reply:
Ok I think that we can delete the `patches` + update the version of `changesets-cli` with only small chance of it breaking publishing. The patch allows us to publish private packages, which we no longer do anymore as part of `changesets` (assuming that we don't use changesets to publish chain-mon related software?) If we do use changesets to publish chain-mon, then the following diff should fix the issue:

```diff
diff --git a/packages/chain-mon/package.json b/packages/chain-mon/package.json
index f183602d7..528ee229d 100644
--- a/packages/chain-mon/package.json
+++ b/packages/chain-mon/package.json
@@ -1,5 +1,4 @@
 {
-  "private": true,
   "name": "@eth-optimism/chain-mon",
   "version": "0.5.7",
   "description": "[Optimism] Chain monitoring services",
```

Reply:
Yea this patchfile is hacky and it would be good to get rid of it.

The best way to get rid of it is to just enable private packages. [Docs are here](https://github.com/changesets/changesets/blob/main/docs/versioning-apps.md)

Reply:
Thanks, I've clarified the README text in 1b564fbff to indicate this patch is no longer needed, and opened https://github.com/ethereum-optimism/client-pod/issues/455 to track it's removal

Reply:
Opened https://github.com/ethereum-optimism/optimism/pull/8963 to address